website/docs/quick-start-guide.md

```diff
@@ -778,18 +727,20 @@ val updates = convertToStringList(dataGen.generateUpdates(10))
 val df = spark.read.json(spark.sparkContext.parallelize(updates, 2))
 df.write.format("hudi").
   options(getQuickstartWriteConfigs).
-  option(PRECOMBINE_FIELD_OPT_KEY, "ts").
-  option(RECORDKEY_FIELD_OPT_KEY, "uuid").
-  option(PARTITIONPATH_FIELD_OPT_KEY, "partitionpath").
+  option(PRECOMBINE_FIELD_NAME.key(), "ts").
+  option(RECORDKEY_FIELD_NAME.key(), "uuid").
+  option(PARTITIONPATH_FIELD_NAME.key(), "partitionpath").
   option(TABLE_NAME, tableName).
   mode(Append).
   save(basePath)
 ```
 :::note
-Notice that the save mode is now `Append`. In general, always use append mode unless you are trying to create the table for the first time.
-[Querying](#query-data) the data again will now show updated trips. Each write operation generates a new [commit](/docs/concepts)
+- Notice that the save mode is now `Append`. In general, always use append mode unless you are trying to create the table for the first time.
+[Querying](#query-data) the data again will now show updated trips. Each write operation generates a new [commit](/docs/next/concepts)
 denoted by the timestamp. Look for changes in `_hoodie_commit_time`, `rider`, `driver` fields for the same `_hoodie_record_key`s in previous commit.
+- Updates to a Keyless table via spark datasource is not supported. Updates for keyless tables are supported with Spark-SQL MergeInto and UPDATE commands.
```

Comment:
Just curious, I guess updates to keyless table via deltaStreamer are also not supported in that case?

Reply:
yes.